frontend/src/container/NewWidget/RightContainer/index.tsx

```diff
@@ -34,14 +38,15 @@ function RightContainer({
 
 	return (
 		<Container>
-			<InputComponent
-				labelOnTop
-				label="Panel Type"
-				size="middle"
-				value={selectedGraphType}
-				disabled
-			/>
-
+			<Title>Panel Type</Title>
+			<Select
+				onChange={(value: ITEMS): void => setGraphHandler(value)}
+				value={selectedGraph}
+				style={{ width: '100%' }}
+			>
+				<Option value="VALUE">Value</Option>
```

Comment:
can we make use of this config `/frontend/src/container/NewDashboard/ComponentsSlider/menuItems.ts`